test/ruby/test_yjit.rb

```diff
@@ -1517,4 +1515,10 @@ def eval_with_jit(script, call_threshold: 1, timeout: 1000, mem_size: nil)
     stats_r&.close
     stats_w&.close
   end
+
+  # A wrapper of EnvUtil.invoke_ruby that uses RbConfig.ruby instead of EnvUtil.ruby
+  # that might use a wrong Ruby depending on your environment.
+  def invoke_ruby(*args, **kwargs)
+    EnvUtil.invoke_ruby(*args, **{ rubybin: RbConfig.ruby, **kwargs })
```

Comment:
```suggestion
    EnvUtil.invoke_ruby(*args, rubybin: RbConfig.ruby, **kwargs)
```
Unless it was intentional?

Reply:
I was initially thinking of using `**{ rubybin: RbConfig.ruby }.merge!(kwargs)` so that `rubybin` can be safely overridden. But then I ended up rewriting that to this for simplifying the code, and now it prints a warning if `rubybin` is provided, just like yours. We shouldn't pass `rubybin` there anyway, so it seems fine to use either of these. Applied your suggestion https://github.com/ruby/ruby/pull/8689/commits/1256de9e12b60c754025cb6118854f5309516488.